### PR TITLE
Added Transmission 3.0.0 Early Preview

### PR DESCRIPTION
**Pull request by owenmurr**
Added Transmission 3.0.0 Early Preview

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1310)

### CODE REVIEW

File: manifests/Transmission/Transmission/3.0.0.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Id: transmission.transmission
+Version: 3.0.0
+Name: Transmission for Windows
+Publisher: Transmission
+License: GNU GENERAL PUBLIC LICENSE
+AppMoniker: Transmission
+Tags: transmission, bittorrent, p2p
```
Comment (MarNicGit) [inline manifests/Transmission/Transmission/3.0.0.yaml:7]:
```suggestion
Tags: transmission, bittorrent, p2p, torrent
```

**Review by latere-a-latere — COMMENTED** [review]
gah, meant to make that a comment instead of a review. Oh well. adding "torrent" as a tag would be a good idea here.

File: manifests/Transmission/Transmission/3.0.0.yaml (hunk 1 above)
Comment (owenmurr) [reply manifests/Transmission/Transmission/3.0.0.yaml]: Sorted. Thank you.

**Review by ghost — APPROVED** [review]
Validation has completed

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**